### PR TITLE
Small doc fixes

### PR DESCRIPTION
**Pull request by mcarpenter**
Hi,

Just a handful of typos and a small update to the PythonHack flow notes.

### CODE REVIEW

**Comment by mcarpenter** [comment]
I signed it!

File: admin.adoc [hunk 1]
```diff
@@ -861,15 +861,12 @@ the following command line.
 grr_config_updater upload_python --file=myfile.py --platform=windows
 ------------------------------------------------------------------------
 
-At the end of this you should see something like:
-------------------------------------------------------------------------
-Uploaded successfully to aff4:/config/python_hacks/myfile.py
```
Comment (mbushkov) [inline admin.adoc]: Just checking - is this message not shown or do you think this information is of no use?

Comment (mcarpenter) [inline admin.adoc]: This message is no longer shown (and I couldn't find "Uploaded successfully" in HEAD).

There is a message regarding the config file that was used but that didn't seem particularly useful in the context of this documentation.

Comment (mbushkov) [inline admin.adoc]: It should be "Uploaded to aff4:/config/python_hacks/myfile.py" (per https://github.com/google/grr/blob/e4d6919be3ce631a123dfe7f4aa1a4769d7ec2e2/grr/lib/maintenance_utils.py#L92).
But apparently this message is not shown unless "--verbose" flag is provided. Probably fine to remove this part.

**Pull request merged**